### ISSUE TITLE
SwiftUI support plans?

### ISSUE THREAD

**Issue by hasanals** [flowkey/UIKit-cross-platform#326]
Hi,

I was wondering if you are planning to add swiftui support to flowkey?

Thanks,

**Comment by ephemer** [comment]
@hasanals we would love to but it'd be a huge project and we need to prioritise other projects for the time being.

There are some other open source SwiftUI projects that we or someone else might be able to integrate with our open source UIKit. Alternatively projects like Tokamak have an experimental GTK+ rendering backend which may be more efficient than trying to send SwiftUI->UIKit->OpenGL, but it's at an early stage and is not the focus of their team currently AFAIK.

In short, it would be nice, but the answer is no from us for the foreseeable future. I will close this issue now because I don't think there's more to discuss on it at the moment
